Hi,<br>I&#39;ve installed the latest snapshot taken from svn developer&#39;s trunk but I had the same problems. This is my configuration:<br><ul><li>Ubuntu 2.6.38-8 kernel</li><li>Openssh_5.8p1 openssl 0.9.8o</li><li>Libtool version 2.4</li>
<li>Open mpi 1.7 rc5 and latest snapshots. </li></ul>Do you think my problem could be related with the operating system used or with any parameter or configuration? I&#39;ve also checked the ssh log file but I didn&#39;t find any problem.<br>
Thanks in advance<br>Ada<br><br><br><br>Il giorno martedì 22 gennaio 2013, Ralph Castain ha scritto:<br>&gt;<br>&gt; Ouch - no, you&#39;d have to take it from the developer&#39;s trunk, either via svn checkout or the nightly developer&#39;s snapshot<br>
&gt;<br>&gt; On Jan 22, 2013, at 12:35 PM, Ada Mancuso &lt;<a href="mailto:mancuso.ada@gmail.com">mancuso.ada@gmail.com</a>&gt; wrote:<br>&gt;<br>&gt; My problem is that I have to use openmpi 1.7 rc5 because I&#39;m using the Java binding mpijava... Is it present in the latest snapshot you told me? If so where can I find it?<br>
&gt; Thanks a lot<br>&gt; Ada<br>&gt;<br>&gt; Il giorno 22/gen/2013 21:03, &quot;Ralph Castain&quot; &lt;<a href="mailto:rhc@open-mpi.org">rhc@open-mpi.org</a>&gt; ha scritto:<br>&gt;&gt;<br>&gt;&gt; It seems to be working fine for me with the latest 1.7 tarball (not rc5 - I didn&#39;t test that one). Could be there was a problem that has since been fixed. We are getting ready to release an updated rc, so you might want to try it (or use the latest nightly 1.7 snapshot).<br>
&gt;&gt;<br>&gt;&gt;<br>&gt;&gt; On Jan 22, 2013, at 9:57 AM, Ada Mancuso &lt;<a href="mailto:mancuso.ada@gmail.com">mancuso.ada@gmail.com</a>&gt; wrote:<br>&gt;&gt;<br>&gt;&gt; Hi,<br>&gt;&gt; I&#39;m trying to run my mpi program using open mpi 1.7 rc5 on 4 machines using the command:<br>
&gt;&gt; mpirun -np4 -hostfile file a.out<br>&gt;&gt; but i get the following message errors:<br>&gt;&gt; ORTE_ERROR_LOG: A message is attempting to be sent to a process whose contact information is unknown in file ../../../../../ompi/orte/mca/rml/oob/rml_oob_send.c<br>
&gt;&gt; attempted to send to [[21341,0],2]: tag 15 <br>&gt;&gt; ORTE_ERROR_LOG: A message is attempting to be sent to a process whose contact information is unknown in file ../../../../ompi/orte/mca/grpcomm/base/grpcomm_base_xcast.c<br>
&gt;&gt; The file etc/hosts is composed by ipaddress hostname, I have exchange ssh keys among the machines and ssh login works without requiring authentication password. Surprisingly if I try to run my program with at most 2 hosts, and so the file hosts contains only two hosts, it works but if i try to run my program with more than two hosts i have this error; mpi works well on each machine and I also tried to run my program with different couple of machines in order to be sure that no machine could be the problem.<br>
&gt;&gt; Can you help me please?<br>&gt;&gt; Ada<br>&gt;&gt; _______________________________________________<br>&gt;&gt; users mailing list<br>&gt;&gt; <a href="mailto:users@open-mpi.org">users@open-mpi.org</a><br>&gt;&gt; <a href="http://www.open-mpi.org/mailman/listinfo.cgi/users">http://www.open-mpi.org/mailman/listinfo.cgi/users</a><br>
&gt;&gt;<br>&gt;&gt;<br>&gt;&gt;<br>&gt;&gt; _______________________________________________<br>&gt;&gt; users mailing list<br>&gt;&gt; <a href="mailto:users@open-mpi.org">users@open-mpi.org</a><br>&gt;&gt; <a href="http://www.open-mpi.org/mailman/listinfo.cgi/users">http://www.open-mpi.org/mailman/listinfo.cgi/users</a><br>
&gt;<br>&gt; _______________________________________________<br>&gt; users mailing list<br>&gt; <a href="mailto:users@open-mpi.org">users@open-mpi.org</a><br>&gt; <a href="http://www.open-mpi.org/mailman/listinfo.cgi/users">http://www.open-mpi.org/mailman/listinfo.cgi/users</a><br>
&gt;<br>&gt;<br>

